packages/v4/patternfly-docs/content/training/html-css/css-variables-and-overrides.md

```diff
@@ -0,0 +1,227 @@
+---
+id: HTML CSS variables and overrides training
+section: training
+hideNavItem: true
+---
+import { Button, ClipboardCopy } from '@patternfly/react-core';
+
+# CSS variables and overrides
+PatternFly Core is based on the principles of <a href="http://bradfrost.com/blog/post/atomic-web-design/" target="_blank">Atomic design</a> and BEM (Block, Element, Modifier). <a href="http://getbem.com/introduction/" target="_blank">BEM</a> is a popular CSS methodology for building modular, scalable applications. It provides scope, avoids inheritance, and reduces CSS specificity and conflicts.
```

Comment:
```suggestion
PatternFly Core is based on the principles of <a href="http://bradfrost.com/blog/post/atomic-web-design/" target="_blank">Atomic Design</a> and <a href="http://getbem.com/introduction/" target="_blank">BEM</a> Block, Element, Modifier). BEM is a popular CSS methodology for building modular, scalable applications. It provides scope, avoids inheritance, and reduces CSS specificity and conflicts.
```

small technicalities - it seems like Atomic Design refers to a book & is a proper noun, so it should be capitalized. I would also go ahead and link to BEM the first time it's mentioned.